packages/rocketchat-file-upload/lib/FileUploadBase.js

```diff
@@ -18,6 +18,10 @@ UploadFS.config.defaultStorePermissions = new UploadFS.StorePermissions({
 			return true;
 		}
 
+		if (RocketChat.authz.canAccessRoom(doc)) {
```

Comment:
same as before.. since `canAccessRoom` already expects a `room` and `user` parameters.. I would change this to  `RocketChat.authz.canAccessRoom(null, null, doc)`

Reply:
Fixed.